avail-js/examples/node-examples/src/tx-balance-transfer.ts

```diff
@@ -11,19 +11,33 @@
 import { getDecimals, initialize, formatNumberToBalance, getKeyringFromSeed, isValidAddress } from "avail-js-sdk"
 import { ISubmittableResult } from "@polkadot/types/types/extrinsic"
 import { H256 } from "@polkadot/types/interfaces/runtime"
-
+import {  GenericExtrinsicEra as ExtrinsicEra, TypeRegistry  } from '@polkadot/types';
 import config from "../../config"
 
 const main = async () => {
   try {
     if (!isValidAddress(config.recipient)) throw new Error("Invalid Recipient")
 
     const api = await initialize(config.endpoint)
+    const meta = api.runtimeMetadata;
+    console.log(meta.toHex())
```

Comment:
This looks like something that was used to debug the metadata but was forgotten to be removed.